tests/databases/api_tests.py

```diff
@@ -1668,38 +1675,74 @@ def test_validate_parameters_invalid_payload_schema(self):
         response = json.loads(rv.data.decode("utf-8"))
 
         assert rv.status_code == 422
-        assert response == {
-            "errors": [
-                {
-                    "message": "Missing data for required field.",
-                    "error_type": "INVALID_PAYLOAD_SCHEMA_ERROR",
-                    "level": "error",
-                    "extra": {
-                        "invalid": ["engine"],
-                        "issue_codes": [
-                            {
-                                "code": 1020,
-                                "message": "Issue 1020 - The submitted payload has the incorrect schema.",
-                            }
-                        ],
+        invalid_schema = response["errors"][0]["extra"]["invalid"][0]
+        # This is done because this array of errors does not have a deterministic order
+        if invalid_schema == "engine":
```

Comment:
@betodealmeida and @hughhhh I was unable to find a way of sorting this array of objects, despite reading about it. Part of the issue is that the only thing that is different between two objects is an array inside of an object. I hate this methodology though. How do I make it better?